### PR TITLE
tutorial: Remove kind abbreviation syntax

### PR DESCRIPTION
**Pull request by tchajed**
Fixes \#975.

### CODE REVIEW

**Comment by tchajed** [comment]
I left in `<>` in the descriptions of how `as_requires` and `as_ensures` are encoded, to avoid introducing the slightly more awkward `=!=` or using `~(y==y')` everywhere. Even these should probably be fixed - it really is less confusing if the tutorial is all valid F* as much as possible, or is obviously a different syntax.

**Comment by tchajed** [comment]
I left `<>` in the description of the `as_requires` and `as_ensures` encoding because I didn't want to introduce `=!=` and `~(y==y')` seemed awkward. Nonetheless perhaps we should switch to one of those - it's much less confusing if the whole tutorial uses valid F* code, or otherwise uses an obviously different syntax.

File: doc/tutorial/tutorial.mdk [hunk 1]
```diff
@@ -3270,9 +3259,9 @@ following form---they transform pure post-conditions to pure
 pre-conditions.
 
 ```
-kind PurePost (a:Type) = a -> Type
-kind PurePre = Type
-kind PureWP (a:Type) = PurePost a -> PurePre
+let pure_post (a:Type) = a -> Type
+let pure_pre = Type
+let pure_wp (a:Type) = pure_post a -> pure_pre
```
Comment (cpitclaudel) [inline doc/tutorial/tutorial.mdk]: Would this be more readable with `type pure_post …` instead of `let pure_post …`?

**Comment by msprotz** [comment]
This looks good to me. Thanks Tej!

**Pull request merged**